plugin/trino-kafka-event-listener/src/main/java/io/trino/plugin/eventlistener/kafka/KafkaEventListenerConfig.java

```diff
@@ -0,0 +1,211 @@
+    private boolean tolerateStartupFailure;
+    private Optional<String> environmentVariablePrefix = Optional.empty();
+
+    public boolean isAnonymizationEnabled()
+    {
+        return anonymizationEnabled;
+    }
+
+    @Config("kafka-event-listener.anonymization.enabled")
+    public KafkaEventListenerConfig setAnonymizationEnabled(boolean anonymizationEnabled)
+    {
+        this.anonymizationEnabled = anonymizationEnabled;
+        return this;
+    }
+
+    public String getBrokerEndpoints()
+    {
+        return brokerEndpoints;
+    }
+
+    @Config("kafka-event-listener.broker-endpoints")
+    public KafkaEventListenerConfig setBrokerEndpoints(String brokerEndpoints)
+    {
+        this.brokerEndpoints = brokerEndpoints;
+        return this;
+    }
+
+    public Optional<String> getClientId()
+    {
+        return clientId;
+    }
+
+    @Config("kafka-event-listener.client-id")
+    public KafkaEventListenerConfig setClientId(String clientId)
+    {
+        this.clientId = Optional.ofNullable(clientId);
+        return this;
+    }
+
+    public Optional<String> getCompletedTopicName()
+    {
+        return completedTopicName;
+    }
+
+    @Config("kafka-event-listener.completed-event.topic")
+    public KafkaEventListenerConfig setCompletedTopicName(String completedTopicName)
+    {
+        this.completedTopicName = Optional.ofNullable(completedTopicName);
+        return this;
+    }
+
+    public Optional<String> getCreatedTopicName()
+    {
+        return createdTopicName;
+    }
+
+    @Config("kafka-event-listener.created-event.topic")
+    public KafkaEventListenerConfig setCreatedTopicName(String createdTopicName)
+    {
+        this.createdTopicName = Optional.ofNullable(createdTopicName);
+        return this;
+    }
+
+    @ConfigDescription("Whether to publish io.trino.spi.eventlistener.QueryCreatedEvent")
+    @Config("kafka-event-listener.publish-created-event")
+    public KafkaEventListenerConfig setPublishCreatedEvent(boolean publishCreatedEvent)
+    {
+        this.publishCreatedEvent = publishCreatedEvent;
+        return this;
+    }
+
+    public boolean getPublishCreatedEvent()
+    {
+        return publishCreatedEvent;
+    }
+
+    @ConfigDescription("Whether to publish io.trino.spi.eventlistener.QueryCompletedEvent")
+    @Config("kafka-event-listener.publish-completed-event")
+    public KafkaEventListenerConfig setPublishCompletedEvent(boolean publishCompletedEvent)
+    {
+        this.publishCompletedEvent = publishCompletedEvent;
+        return this;
+    }
+
+    public boolean getPublishCompletedEvent()
+    {
+        return publishCompletedEvent;
+    }
+
+    @ConfigDescription("Comma-separated list of field names to be excluded from the Kafka event (their value will be replaced with null). E.g.: 'payload,user'")
+    @Config("kafka-event-listener.excluded-fields")
+    public KafkaEventListenerConfig setExcludedFields(String excludedFields)
```

Comment:
Now airlift supports passing a `List<String>` directly so we don't need a splitter to split them